Sources/BraveWallet/Crypto/Transaction Confirmations/SwapTransactionConfirmationView.swift

```diff
@@ -0,0 +1,299 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
its bunny year now :)

Reply:
Fixed in [1c4138a](https://github.com/brave/brave-ios/pull/6826/commits/1c4138a1cc888815116a3bcfece0711369eb5233)